lib/active_fedora/file.rb

```diff
@@ -139,6 +140,19 @@ def metadata?
       false
     end
 
+    def external_content?
+      external_url.present?
+    end
+
+    def external_url
+      return nil unless mime_type.start_with? "message/external-body"
+      mime_type[/url=['"](.+?)['"]/i, 1]
+    end
+
+    def external_url=(external_file_url)
+      self.mime_type = "message/external-body; access-type=URL; URL=\"#{external_file_url}\""
```

Comment:
@awead I'm mostly thinking about someone having `message/external-body; access-type="local-file"; local-file="/path/to/binary"` — I'm not sure what we should do with that, and I think it would be best to be tolerant in what we allow people to do until we have a better handle on that.